BlueprintUI/Tests/EnvironmentTests.swift

```diff
@@ -0,0 +1,98 @@
+//
+//  EnvironmentTests.swift
+//  BlueprintUI
+//
+//  Created by Kyle Van Essen on 3/28/20.
+//
+
+import Foundation
+import XCTest
+@testable import BlueprintUI
+
+
+class EnvironmentTests : XCTestCase {
```

Comment:
This doesn't do a whole bunch right now, was mostly for me setting breakpoints while writing.